doc/news/changes.h

```diff
@@ -201,6 +201,13 @@ inconvenience this causes.
 <h3>Specific improvements</h3>
 
 <ol>
+ <li> New: Added an optional string parameter to the ParameterHandler::read_ functions.
```

Comment:
doxygen is going to complain about not being able to find that function. Just list all of them.